functionaltests/utils_test.go

```diff
@@ -59,12 +60,36 @@ func copyTerraforms(t *testing.T) {
 	require.NoError(t, err)
 }
 
+type deploymentOutput struct {
```

Comment:
maybe 
```suggestion
type deploymentInfo struct {
```
would be a better name? Output is coupled with Terraform but this does not represent a generic "output"

Reply:
Renamed in [11e29e3](https://github.com/elastic/apm-server/pull/16418/commits/11e29e3c9cfea6811a8491c2ef6bcd1d968a9e7b).